packages/contentstack-launch/env-unit-tests-sample.txt

```diff
@@ -0,0 +1,3 @@
+ENVIRONMENT=<DEPLOYMENT_ENVIRONMENT>
+ORG=<ORG_UID>
+PROJECT=<PROJECT_NAME>
```

Comment:
Can we rename this file as `example.env`?

Reply:
renamed